RELEASING.md

```diff
@@ -19,9 +24,13 @@
   - make sure the PR title starts with `rel:` to conform with our semantic commits convention
   - add the `no-changelog`, `type: maintenance`, and `version: no bump` labels
 - Once the pull request is approved, squash-merge it
+
+## Git Tags and Releases
 - Checkout `main` and fetch the now-updated `main` branch.
-- Apply a tag for the new version on the merged commit (e.g. `git tag -a v2.3.1 -m "v2.3.1"`)
-- Push the tag upstream (this will kick off the release pipeline in CI) e.g. `git push origin v2.3.1`
+- Apply a tag for the new version on the merged commit (e.g. `git tag -a v2.3.1 -m "web-distro-v2.3.1"`). The tag MUST begin with either `web-distro-v` or `react-instrumentation-v` in order to trigger our CI release pipeline
```

Comment:
should the -a also have the new prefix?

Reply:
whoops yeah i put it in the wrong spot 🙈